Makefile

```diff
@@ -15,7 +15,10 @@
 # Various versions - Minishift, default OpenShift, default B2D ISO
 MINISHIFT_VERSION = 1.0.1
 OPENSHIFT_VERSION = v1.5.1
-ISO_VERSION = v1.0.2
+B2D_ISO_VERSION = v1.0.2
+CENTOS_ISO_VERSION = v1.0.0
+DOC_VARIABLES = '-e OPENSHIFT_VERSION=$(OPENSHIFT_VERSION) -e MINISHIFT_VERSION=$(MINISHIFT_VERSION) \
```

Comment:
Let's move this variable under the comment "# Setup for the docs tasks". The variables are somewhat grouped together